apps/src/templates/studioHomepages/Incubator.jsx

```diff
@@ -2,8 +2,36 @@ import React, {Component} from 'react';
 import HeaderBanner from '../HeaderBanner';
 import TwoColumnActionBlock from '@cdo/apps/templates/studioHomepages/TwoColumnActionBlock';
 import Button from '@cdo/apps/templates/Button';
+import DCDO from '@cdo/apps/dcdo';
+import i18n from '@cdo/locale';
+import {getStore} from '@cdo/apps/redux';
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
+import {EVENTS, PLATFORMS} from '@cdo/apps/lib/util/AnalyticsConstants';
+import {LmsLinks} from '@cdo/apps/util/sharedConstants';
 
 class Incubator extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      canvasBlockEnabled: DCDO.get('incubator-canvas-block-enabled', true),
+      currentUser: getStore().getState().currentUser,
+    };
+  }
+
+  reportEvent(eventName, platform = PLATFORMS.STATSIG) {
+    analyticsReporter.sendEvent(
+      eventName,
+      {
+        userId: this.state.currentUser.userId,
+        userType: this.state.currentUser.userType,
+        userName: this.state.currentUser.userName,
+        userRoleInCourse: this.state.currentUser.userRoleInCourse,
```

Comment:
The way Statsig is configured, it [builds a user](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/lib/util/StatsigReporter.js#L23) object as part of the initialization process and includes it in the event log, so you don't need to add it here in the metadata. Also, I'm not sure we want to log the username, as I believe this is PII.

As a result, I'd remove everything here except userRoleInCourse.

Reply:
Got it, thanks!